### PR TITLE
Fmendez azure install juniper marketplace private

### PR DESCRIPTION
**Pull request by mendezcirera**
- Updated the documentation to deploy the SSN software using the Juniper branded images in the Azure marketplace.
- Removed references to the 128 Technology marketplace and its images.

### CODE REVIEW

File: docs/intro_installation_azure.md [hunk 1]
```diff
@@ -571,74 +454,16 @@ and paste the following JSON content, please adjust the values to your specific
...
+Go to the **Session Smart Networking** offering following the steps described in the section "Selecting the Azure Image". Click on the "Get it now" button, agree to the terms of use and privacy policy of the image, click on the "Get started" button to enable programmatic deployment for the subscription and click the button "Save" to save the changes.
 
 ![Plans](/img/platforms_azure_programmatically.png)
 
-Close the "Configure Programmatic Deployment" window, click on the tab "Plans" and lastly copy to the clipboard the URL of the "Standalone Conductor" template located in the field "URL". Lastly launch the template running the following command:
+Close the "Configure Programmatic Deployment" window, click on the tab "Plans + Pricing" and lastly copy to the clipboard the URL of the "Conductor" template located in the field "URL". Lastly launch the template running the following command:
```
Comment (Chr1st0ph3rTurn3r) [inline docs/intro_installation_azure.md:461]: Close the "Configure Programmatic Deployment" window.
Click on the tab "Plans + Pricing".
Copy the URL of the "Conductor" template located in the field "URL" to the clipboard. 
Launch the template running the following command:

**Review by Chr1st0ph3rTurn3r — APPROVED** [review]

**Review by BenMatase — APPROVED** [review]

**Review by agrawalkaushik — APPROVED** [review]

File: docs/intro_installation_azure.md [hunk 2]
```diff
@@ -19,22 +19,65 @@ The following infrastructure must exist in your Azure subscription:
...
+* Please click [here](https://portal.azure.com) to go to the Azure portal.
+* On the search box right at the top of the screen, search for "Subscriptions". If you cannot find the subscription associated with your Azure account click [here](https://portal.azure.com/#blade/Microsoft_Azure_Billing/SubscriptionsBlade).
+* Take note of your Subscription ID.
+
+2. Contact your Juniper Networks Sales representative and provide:
+
+* The Subscription ID of the Azure account that will be used for the deployment.
+* The version of the Session Smart Networking software. Your Juniper Sales representative will assist you if you don't know the version you need for your deployment.
+
+3. Wait for the confirmation from your Juniper Sales representative to confirm that your Azure Subscription has been allowlisted and therefore access has been granted.
```
[the remark this answers is not in the document]
Comment (mendezcirera) [reply docs/intro_installation_azure.md:55]: Chris updated the document before, so I followed the same approach. My understanding is that this is in accordance to our new policy

File: docs/intro_installation_azure.md [hunk 3]
```diff
@@ -19,22 +19,65 @@ The following infrastructure must exist in your Azure subscription:
...
+
+4. Once your Subscription ID has been allowlisted, validate that the Private image has been shared with your Subscription:
+
+* Click [here](https://portal.azure.com) to go to the Azure portal. On the search box right at the top of the screen, search for "Marketplace". If you are unable to access the Marketplace via your Azure Portal, click [here](https://portal.azure.com/#blade/Microsoft_Azure_Marketplace/GalleryMenuBlade/selectedMenuItemId/home).
```
[the remark this answers is not in the document]
Comment (mendezcirera) [reply docs/intro_installation_azure.md:59]: I opted for describing the process step by step first, and then provide direct links if the user can't find the resource in the Azure portal.

File: docs/intro_installation_azure.md [hunk 4]
```diff
@@ -19,22 +19,65 @@ The following infrastructure must exist in your Azure subscription:
...
+* The following banner is displayed at the top:
+
+![Marketplace private offerings](/img/platforms_azure_marketplace_private_banner.png)
+
+* Click on the "View private products" link.
+* If the Private image of the **Session Smart Networking** offering is displayed, then the Private image has been shared successfully with your Azure Subscription ID.
+
+<img src="/img/platforms_azure_marketplace_image_private.png" alt="Session Smart Networking Private offering" width="192" height="243" />
+
+On the contrary, if you prefer not to use the Private image then refer to the Public image of the **Session Smart Networking** offering available [here](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/juniper-networks.session-smart-networking-payg?tab=Overview).
```
[the remark this answers is not in the document]
Comment (mendezcirera) [reply docs/intro_installation_azure.md:69]: It does, in fact I left that section almost ready to go, with placeholders to split it into subsections accordingly, but I was thinking to do that once we have the Public BYOL image :)

File: docs/intro_installation_azure.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Chr1st0ph3rTurn3r) [reply docs/intro_installation_azure.md:55]: I believe that term came from someone above my pay grade, and I think I looked into it then as well. I would not have left it that way if it was not. I can't remember exactly, but I'm pretty sure it was an Azure term.

**Review by MichaelBaj — APPROVED** [review]

**Pull request merged**